src/casp/services/db/migrations/versions/t_2024-03-13_14-58-14_non_unique_model_path.py

```diff
@@ -0,0 +1,27 @@
+"""non unique model path
+
+Revision ID: a2e0ae902a06
+Revises: 2633d266e26d
+Create Date: 2024-03-13 14:58:14.174652
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a2e0ae902a06"
+down_revision = "2633d266e26d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("cas_model_model_file_path_key", "cas_model", type_="unique")
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint("cas_model_model_file_path_key", "cas_model", ["model_file_path"])
```

Comment:
This could potentially fail if there is data within the table when we try to downgrade that violates this constraint.  Do we want to have some way of handling that here, or would it better to let it fail so whoever is running this knows they have to manually make some data changes?